packages/libraries/cli/package.json

```diff
@@ -30,11 +30,11 @@
     "graphql"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --build --clean && tsc --build",
+    "build:watch": "tsc --build --watch",
     "oclif:pack": "npm pack && pnpm oclif pack tarballs --no-xz",
     "oclif:upload": "pnpm oclif upload tarballs --no-xz",
     "postpack": "rm -f oclif.manifest.json",
-    "prebuild": "rimraf dist",
```

Comment:
minor: I prefer using pre- scripts in this case.
`"prebuild": : "tsc --build --clean"`

Reply:
Whoops, missed this, will make a follow up PR.

Reply:
https://github.com/graphql-hive/console/pull/6331